notesapi/v1/tests/test_views.py

```diff
@@ -47,51 +44,66 @@ def setUp(self):
             ],
         }
 
-        self.expected_note = {
-            "created": "2014-11-26T00:00:00+00:00",
-            "updated": "2014-11-26T00:00:00+00:00",
-            "user": TEST_USER,
-            "usage_id": "test-usage-id",
-            "course_id": "test-course-id",
-            "text": "test note text",
-            "quote": "test note quote",
-            "ranges": [
-                {
-                    "start": "/p[1]",
-                    "end": "/p[1]",
-                    "startOffset": 0,
-                    "endOffset": 10,
+    def tearDown(self):
+        for note_id in note_searcher.all().values_list('id'):
+            get_es().delete(
+                index=settings.ES_INDEXES['default'],
+                doc_type=NoteMappingType.get_mapping_type_name(),
+                id=note_id[0][0]
+            )
+        get_es().indices.refresh()
```

Comment:
would it better to only refresh the specific index being used?  (same question goes for all the occurrences in this PR, and i suspect it would make sense to implement a convenience method to do so)